test/externalTests/anvil.js

```diff
@@ -4,7 +4,7 @@ module.exports = () => async (bot) => {
   const mcData = require('minecraft-data')(bot.version)
   const Item = require('prismarine-item')(bot.version)
   const renameCost = () => mcData.isNewerOrEqualTo('1.8.9') ? 0 : 1 // weird quirk of anvils
-  const renameName = (name) => mcData.isOlderThan('1.14') ? name : JSON.stringify({ text: name }) // weird quirk of anvils
+  const renameName = (name) => mcData.isOlderThan('1.13.2') ? name : JSON.stringify({ text: name }) // weird quirk of anvils
```

Comment:
that doesn't seem to make sense
they made the change between 1.13.1 and 1.13.2 ?

Reply:
this works other didnt is all i can say 🤷